RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -469,7 +472,7 @@ bool DAClusterizerInZ_vect::merge(vertex_t& y, track_t& tks, double& beta) const
     return false;
 
   // merge the smallest distance clusters first
-  std::vector<std::pair<double, unsigned int> > critical;
+  std::vector<std::pair<double, unsigned int>> critical;
```

Comment:
Seems like a bunch of the changes to this file are simply the deletion of the empty space in between the closing brackets.  Is that an intentional style fix?  Otherwise, I would leave it the way it was.

Reply:
makes sense. Thanks, Matti